### DISCUSSION TITLE
Is scrobbling history exportable?

### DISCUSSION THREAD

**Discussion by zblesk** [navidrome/navidrome#3093] [Q&A] [answered]
I've been using Navidrome for some time. 
I've recently installed Maloja as well, for stats; and I use Multi-scrobbler to get scrobble data from Navidrome (and Spotify) to Maloja. 

That seems to work well, but only for new scrobbles. 

Is there any way to get a history out of navidrome, so I could import that as well?

**Comment by deluan** [comment, marked as answer]
Navidrome does not keep a record of all scrobbles. When receiving a scrobble, it only updates the last played and play counts of each song/album/artist and forwards the scrobble to the configured service (Last.fm or ListenBrainz).

Your best bet would be to export it directly from Last.fm, or maybe try to find/write a script that imports from Last.fm directly into Maloja
